src/main/java/com/robomwm/prettysimpleshop/feature/ShowoffItem.java

```diff
@@ -82,7 +80,7 @@ private void saveCache()
         }
     }
 
-    @EventHandler
+    @EventHandler(ignoreCancelled = true, priority = EventPriority.MONITOR)
```

Comment:
ChunkLoadEvent isn't cancelable so the first param is useless. Have you been able to determine what's going on here? And since the plugin is doing stuff to the chunk (albeit asynchronously) it really should be at highest priority at the max instead of monitor.

Reply:
you can change if you want